internal/controller/pkg/revision/runtime_function.go

```diff
@@ -116,7 +120,14 @@ func (h *FunctionHooks) Post(ctx context.Context, pkg runtime.Object, pr v1.Pack
 	}
 
 	sa := build.ServiceAccount()
-	d := build.Deployment(sa.Name, functionDeploymentOverrides(functionMeta, pr)...)
+
+	// determine the function's image, taking into account the default registry
```

Comment:
Nit: I'd prefer if comments were written with proper sentence grammar, i.e. caps and periods. We haven't been super consistent on this but I believe it's the predominant style in the codebase.